ripser/ripser.py

```diff
@@ -37,6 +37,82 @@
 from pyRipser import doRipsFiltrationDMSparse as DRFDMSparse
 
 
+def get_greedy_perm_pc(X, N = -1, metric="euclidean"):
```

Comment:
I think `N=None` is a better convention for 'calculate a reasonable default'. This is debatable. Is there a reason for -1?